scripts/run-integration-tests-npm.sh

```diff
@@ -161,8 +173,54 @@ function setup_node() {
   cd "${cwd}"
 }
 
+function create_npm_package() {
+  local lib="${1}"
+  yarn workspace @hoprnet/${lib} pack
```

Comment:
Why don't add a `yarn workspace @hoprnet/${lib} prepublishOnly` to simulate npm behavior?
(not important and can be done later)